<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebuggerWindow</class>
 <widget class="QMainWindow" name="DebuggerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCSX2 Debugger</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icons/AppIcon64.png</normalon>
   </iconset>
  </property>
  <widget class="QToolBar" name="toolBarDebug">
   <property name="windowTitle">
    <string>Debug</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
   <addaction name="actionStepInto"/>
   <addaction name="actionStepOver"/>
   <addaction name="actionStepOut"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAnalyse"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionStepInto"/>
    <addaction name="actionStepOver"/>
    <addaction name="actionStepOut"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionOnTop"/>
   </widget>
   <widget class="QMenu" name="menuLayouts">
    <property name="title">
     <string>Layouts</string>
    </property>
    <addaction name="actionResetAllLayouts"/>
    <addaction name="actionResetDefaultLayouts"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuDebug"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindows"/>
   <addaction name="menuLayouts"/>
  </widget>
  <widget class="QToolBar" name="toolBarView">
   <property name="windowTitle">
    <string>View</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOnTop"/>
  </widget>
  <widget class="QToolBar" name="toolBarSystem">
   <property name="windowTitle">
    <string>System</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShutDown"/>
   <addaction name="actionReset"/>
  </widget>
  <action name="actionRun">
   <property name="icon">
    <iconset theme="play-line"/>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionStepInto">
   <property name="icon">
    <iconset theme="debug-step-into-line"/>
   </property>
   <property name="text">
    <string>Step Into</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionStepOver">
   <property name="icon">
    <iconset theme="debug-step-over-line"/>
   </property>
   <property name="text">
    <string>Step Over</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionStepOut">
   <property name="icon">
    <iconset theme="debug-step-out-line"/>
   </property>
   <property name="text">
    <string>Step Out</string>
   </property>
   <property name="shortcut">
    <string>Shift+F11</string>
   </property>
  </action>
  <action name="actionOnTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="pin-filled"/>
   </property>
   <property name="text">
    <string>Always On Top</string>
   </property>
   <property name="toolTip">
    <string>Show this window on top</string>
   </property>
  </action>
  <action name="actionAnalyse">
   <property name="icon">
    <iconset theme="magnifier-line"/>
   </property>
   <property name="text">
    <string>Analyze</string>
   </property>
  </action>
  <action name="actionResetAllLayouts">
   <property name="text">
    <string>Reset All Layouts</string>
   </property>
  </action>
  <action name="actionResetDefaultLayouts">
   <property name="text">
    <string>Reset Default Layouts</string>
   </property>
  </action>
  <action name="actionResetSplitterPositions">
   <property name="text">
    <string>Reset Splitter Positions</string>
   </property>
  </action>
  <action name="actionShutDown">
   <property name="icon">
    <iconset theme="shut-down-line"/>
   </property>
   <property name="text">
    <string>Shut Down</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset theme="restart-line"/>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="door-open-line"/>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
